OrbitCore/FramePointerValidatorService.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "FramePointerValidatorService.h"
+
+#include <capstone/capstone.h>
+
+#include <vector>
+
+#include "FunctionFramePointerValidator.h"
+#include "OrbitFunction.h"
+#include "OrbitModule.h"
+#include "OrbitProcess.h"
+#include "Pdb.h"
+
+FramePointerValidatorService::FramePointerValidatorService(
+    const ProcessList* process_list, TransactionService* transaction_service)
+    : process_list_{process_list}, transaction_service_{transaction_service} {
+  auto on_request = [this](const Message& msg) { HandleRequest(msg); };
+  transaction_service_->RegisterTransactionRequestHandler(
+      {on_request, Msg_ValidateFramePointer, "Validate Frame Pointers"});
+}
+
+void FramePointerValidatorService::HandleRequest(const Message& message) {
+  // TODO: The code below is about 90% copy&past from SymbolsService.cpp.
```

Comment:
What should we discuss and what is there to do?

Reply:
Ah right, then maybe rephrase to make it clearer that you are suggesting a refactoring to share code with the other `...Service`s.